common_design.theme

```diff
@@ -165,6 +165,17 @@ function common_design_preprocess_html(&$vars) {
   if ($status_code && $status_code->getStatusCode() == 403) {
     $vars['attributes']['class'][] = 'path-error path-error--403';
   }
+
+  // Add a script to detect whether javascript is enabled or not and then
+  // replace the `no-js` class with the `js` one.
+  // Drupal.js does this but way too late on first load causing issues with
+  // css rules that rely on the `js` class being defined or not.
+  $js_detection = [
+    '#tag' => 'script',
+    '#value' => "document.documentElement.className = document.documentElement.className.replace('no-js', typeof document.addEventListener !== 'undefined' ? 'js' : 'no-js');",
```

Comment:
🤦 https://www.drupal.org/project/drupal/issues/2391025

Reply:
In case it's not clear: I support your decision 💯  but really wish we could inline files. The only other option might be to do something like:

```
$js_detection = [
  '#tag' => 'script',
  '#value' => file_get_contents('path/to/file.js'),
];
```

For something so small I don't mind reading inside PHP quotes, but maybe for future if we want to expand this script, we can cheat using this method ☝️

Reply:
Yep, I agree.